### DISCUSSION TITLE
Training on Multiple Subjects (Faces) Resulting in Blended Characteristics in Images

### DISCUSSION THREAD

**Discussion by voidxd** [TheLastBen/fast-stable-diffusion#1584] [Q&A] [answered]
Hi. I'm trying to train 2 subjects with DreamBooth and the output images seem to have mixed characteristics from both subjects, instead of them being very clearly defined as either one subject or the other.

- I'm using 10 images for each. 
![image](https://user-images.githubusercontent.com/58847957/218444203-caf7ecfb-3328-4f90-bced-b5e30e6f3f1a.png)

- I have tried both training them at the same time, and one after the other. From what I've seen in another discussion, [it's better to train them at the same time](https://github.com/TheLastBen/fast-stable-diffusion/discussions/822#discussioncomment-4291607).

- Model: SD 1.5
- The training parameters I've used:
  - UNet_Training_Steps: 2000 (when training at the same time), and 1000 when training them one after the other
  - UNet_Learning_Rate: 5e-6
  - Text_Encoder_Training_Steps: 350
  - Text_Encoder_Learning_Rate: 1e-6

- Output Images
  1. I first trained Aemond from House of the Dragon. 
  ![image](https://user-images.githubusercontent.com/58847957/218445684-e072eff7-3899-40ff-a69c-188b5128c5c5.png)
  2. Next I trained Sandman from the Netflix show. This is where I try to generate something with the keyword `Sandman2022`. You can see StableDiffusion adds that eyepatch that from Aemond.
  ![image](https://user-images.githubusercontent.com/58847957/218446012-efdaa5be-72b3-4751-b0e8-158f45bb8154.png)
  3. Next, I try to see if the training also affected Aemond and it seems it has.
  ![image](https://user-images.githubusercontent.com/58847957/218446308-f8132a93-1b51-4dc7-bcf3-34418ea069ef.png)

Any advice on preventing this from happening would be very much appreciated. Thank you!

**Comment by TheLastBen** [comment, marked as answer]
training two subjects from the same gender always leads to blending, keep the learning rate 2e-6 or below to prevent excessive overfitting, I recommend training each subject on a separate model for maximum quality. and avoid using instance names like "sandman" it is a known word and it reduces the effectiveness of the training.
